prometheusbp/metrics_buckets_spec.go

```diff
@@ -0,0 +1,28 @@
+//go:build metrics_spec
+// +build metrics_spec
+
+package prometheusbp
+
+// DefaultBuckets is the default bucket values for a prometheus histogram metric.
+// These match the spec defined at https://github.snooguts.net/reddit/baseplate.spec/blob/master/component-apis/prom-metrics.md#latency-histograms
+var DefaultBuckets = []float64{
```

Comment:
we also realized that `DefaultBuckets` is a bad name, because this really is just `DefaultLatencyBuckets`, and should not be used on any histogram that's not latency.

I would suggest define the new buckets as `DefaultLatencyBuckets` instead, and make `DefaultBuckets` just an alias to it but also mark it as deprecated:
```go
// Deprecated: Please use DefaultLatencyBuckets instead for latencies, and define your own buckets for non-latency histograms
var DefaultBuckets = DefaultLatencyBuckets
```

Reply:
This sounds like an improvement to me, though without going through all of the uses of DefaultBuckets I'm not sure it needs to be in the same PR.

Reply:
Makes sense, I'll update that. I won't update usage across everything, but I'll at least add a commit to this PR with a find/replace.